app/ui/src/app/integration/detail-page/detail.component.html

```diff
@@ -32,9 +32,9 @@
     </div>
     <div class="col-md-12 details">
       <h1>
-        <syndesis-editable-text [value]="i.name"
+        <syndesis-editable-text [value]="integration.name"
                                 [validationFn]="validateName"
-                                (onSave)="onAttributeUpdated('name', $event)"></syndesis-editable-text>
+                                (onSave)="onAttributeUpdated('name', $event, integration)"></syndesis-editable-text>
```

Comment:
I know this falls outside the scope of this refactor, but just a note for the future: Angular enforces a non `onXXXX` naming convention for output emitter properties. The reason is because Angular transpiles these properties by appending an `on-` to the generated method handler, so this would eventually become `on-onSave`.

Not relevant for this PR but just a word of caution for the future ;)

Reply:
I might as well fix this one now then, tx!